src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -104,6 +110,10 @@ void CreateInteractionChoiceSetRequest::Run() {
       LOG4CXX_ERROR(logger_, "Image verification failed.");
       SendResponse(false, Result::INVALID_DATA);
       return;
+    } else if (verification_result_image == Result::WARNINGS ||
+               verification_result_secondary_image == Result::WARNINGS) {
+      should_send_warnings = true;
```

Comment:
@jacobkeeler that's because this RPC has a specific behaviour.
SDL does not resend any images to HMI for `CreateInteractionChoiceSet` RPC, just sends `VR.AddCommand`, so HMI does not know if images from `CreateInteractionChoiceSet` is valid.
In this specific case, SDL should validate images from `CreateInteractionChoiceSet` on it's side and populate `WARNINGS` result code in case images are not valid. That's why this flag was added here.
For all other RPC`s it`s up to HMI how to process such images and SDL just resends result code and info message from HMI.